### PR TITLE
CLA Cleanup

### PR DESCRIPTION
**Pull request by kingthorin**
Address work-in-progress type comments and content.

Fixes zaproxy/zaproxy#8599

### CODE REVIEW

**Comment by kingthorin** [comment]
I've set DCO to pass and have signed the CLA:

https://github.com/zaproxy/cla/blob/530c2dd37d2f461586f0e7a84181ae2696446d44/signatures/version1/cla.json#L11-L18

**Review by ricekot — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This repo contains the [ZAP](https://github.com/zaproxy/zaproxy/) Contributor Li
 
 For more details see the [ZAP Contributor License Agreement](https://www.zaproxy.org/blog/2024-10-23-zap-contributor-license-agreement/) blog post.
 
-Once https://github.com/zaproxy/zaproxy/pull/8317 has been merged in the main ZAP repo:
+In the main ZAP repos:
 
 * You will be prompted to sign the CLA when you submit your first ZAP PR.
 * If you agree to the CLA then your GitHub username will be recorded in this repo.
-* The [ZAPCLA](ZAPCLA.md) file will be removed from this repo as the one in [zaproxy](https://github.com/zaproxy/zaproxy/) will be the definitive version.
```
Comment (thc202) [inline README.md:11]: This was not done.

Comment (ricekot) [reply README.md:11]: Could be removed in this PR?

**Comment by kingthorin** [comment]
Tweaked.

This PR was done with github.dev so please squash and merge.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**